platform-core-module/src/main/java/com/softicar/platform/core/module/CoreModuleMapper.java

```diff
@@ -0,0 +1,26 @@
+package com.softicar.platform.core.module;
+
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.emf.mapper.IEmfTableRowMapper;
+import com.softicar.platform.emf.table.row.IEmfTableRow;
+import java.util.Optional;
+
+class CoreModuleMapper<T extends IEmfTableRow<T, ?>> implements IEmfTableRowMapper<T, AGCoreModuleInstance> {
```

Comment:
I think this class deserves an explanatory sentence. Hard to grasp what/why this is.

Reply:
honestly I cant really think up a good explanation, Olli can best describe this I think :D